mmm_eval/utils/validation_utils.py

```diff
@@ -0,0 +1,23 @@
+import numpy as np
+import pandas as pd
+
+
+def distribution_to_dataframe(distribution: np.ndarray, index: pd.Index) -> pd.DataFrame:
+    """Generate a DataFrame from a distribution array.
+
+    Args:
+        distribution (np.ndarray): The distribution array, shape (samples x time)
+        index (pd.Index): The index of the actual values, shape (time,)
+
+    Returns:
+        A DataFrame
```

Comment:
nitpick: specify what the rows and columns end up representing (I assume time x samples)